python-lib/dku_aws/aws_command.py

```diff
@@ -28,8 +28,14 @@ def run(self):
         return (cmd, rv, o, e)
 
     def run_and_get_output(self):
-        return self.run()[2]
-    
+        result = self.run()
+        if result[1] != 0:
+            cmd = _convert_to_string(["aws"] + self.args)
+            logging.error('Failed to execute aws command:\n%s\n%s' % (' '.join(cmd), result[3]))
```

Comment:
I don't think it's necessary to log the command again, since the log line just above this one "should" be the command (logged in `run()`).

Also, when I tried it, I had this error (with a python 2.7 code env though):
```
Traceback (most recent call last):
  File "/opt/dataiku-dss-10.0.6/python/dataiku/runnables/server.py", line 45, in serve
    result = runnable.run(report_progress)
  File "/tmp/tmp_folder_NjRbgRoC/dku_code.py", line 54, in run
  File "/data/dataiku/dss_data/plugins/installed/eks-clusters/python-lib/dku_aws/eksctl_command.py", line 36, in run_and_get_output
    logging.error('Failed to execute eksctl command:\n%s\n%s' % (' '.join(cmd), result[3]))
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 21: ordinal not in range(128)
```

I removed the logging of `cmd` from this log line, and the error went away.

That said, I didn't mind having the entire command in the error message on the UI (even the one that Jules shared wasn't too long for me 😅 ).

Reply:
👍 , thanks @marchelleboid  for the review, I will rollback and merge the version that was approved.